modules/stb_vorbis/audio_stream_ogg_vorbis.h

```diff
@@ -41,11 +41,11 @@ class AudioStreamOGGVorbis;
 class AudioStreamPlaybackOGGVorbis : public AudioStreamPlaybackResampled {
```

Comment:
You missed `AudioStreamOGGVorbis()` which has plenty of stuff to move to the header.